back-office/src/components/user/UserLine/UserLine.jsx

```diff
@@ -0,0 +1,37 @@
+//@flow
+import React from 'react'
+import { User } from '../../../user/index'
+import IconButton from 'react-toolbox/lib/button/IconButton'
+
+import './UserLine.css'
+
+class UserLine extends React.Component {
+  
+  props: {
+    user: User,
+    editUser: any,
+    deleteUser: any,
+  }
+
+  handleEdit = () => {
+    this.props.editUser(this.props.user)
+  }
+
+  handleDelete = () => {
+    this.props.deleteUser(this.props.user.id)
+  }
```

Comment:
Instead of having a class, you can have a Functional Component.
For this, you should move the `handleEdit` and `handleDelete` to the container 

```es6
const mapDispatchtoProps = (dispatch, ownProps) => ({
    handleEdit: () => dispatch(editUser(ownProps.user))
    handleDelete: () => dispatch(deleteUser(ownProps.user.id))
})